src/app/Ranking.js

```diff
@@ -0,0 +1,56 @@
+export class Ranking {
+    constructor(mode) {
```

Comment:
Jeśli już zostajemy przy tym includes to najlepiej jak najmniej zagnieżdżać kod. To samo działanie da radę uzyskać poprzez taki zapis:

```
 constructor(mode) {
        const availableModes = ['people', 'starships', 'vehicles']; //ewentualnie nazwa bardziej opisowa
        if (!availableModes.includes(mode)) {
            throw `"${mode}" mode is not avaiable!`;
        }
        this.mode = mode
    }
```

Reply:
Done